gammapy/maps/axes.py

```diff
@@ -1300,8 +1300,12 @@ def from_table(cls, table, format="ogip", idx=0, column_prefix=""):
             if hduclass in {"BKG", "RAD_MAX"} and column_prefix == "ENERG":
                 name = "energy"
 
-            edges_lo = table[f"{column_prefix}_LO"].quantity[0]
-            edges_hi = table[f"{column_prefix}_HI"].quantity[0]
+            edges_lo = table[f"{column_prefix}_LO"].quantity
```

Comment:
While solving the problem, the code here somewhat has the wrong semantics. GADF always stores the IRFs as a single row in a binary table, so to get the first (and until now only) IRF GADF file, `[0]` is correct, so I'd not change this here.

You want to add support for the case that the content of this row is not an array, but a single number. So instead of changing the two lines above and then checking for `ndim > 1`, check for a scalar and upcast if needed:

```
if edges_lo.shape == ():
    edges_lo = edges_lo[np.newaxis]
    edges_hi = edges_hi[np.newaxis]

```
You could also add a warning there.

Simpler would be just doing `edges_lo = np.atleast_1d(table[f"{column_prefix}_LO"].quantity[0])`, but then you can't issue the warning.

Reply:
Good point - I've changed the code and added a warning (although this is now the only place in the entire file where a warning is being issued...)